### DISCUSSION TITLE
스카이박스 변경이 안됩니다.

### DISCUSSION THREAD

**Discussion by Chordcode002** [naverz/zepeto-studio-kor#853] [5. Q&A]
#487 에서 이미 질문이 있긴했습니다만 똑같은 문제인데 해결은 되지 않은 것 같아서 질문드립니다.

관련 코드는 
`RenderSettings.skybox = this.skyboxMaterialNight;`
오직 이 부분 밖에 없습니다. 
단순히 현재 스카이박스를 다른 스카이박스로 바꾸는 코드입니다.

`import {  RenderSettings} from "UnityEngine";` 로 추가해주고

이 this.skyboxMaterialNight는 인스펙터 상에서 Property로 넣어줬습니다.
![Material](https://user-images.githubusercontent.com/42954409/172599948-fd3230c5-52dd-477d-8223-c282b0f55ae5.PNG)
그리고 6면으로 지정된 스카이박스만 되는가 싶어서 그것도 적용해봤습니다만 여전히 똑같은 문제가 발생했습니다.

Error: Exception of type 'ZEPETO.Script.Extensions.ForbiddenTypeExtension+ForbiddenException' was thrown.

다음과 같은 에러메시지가 출력됩니다.

해당 질문 이외에 어떠한 관련 질문도 없어서... 질문드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요,  현재 RenderSetting 접근은 막아 놓은 상태이며 추후에 지원할 수 있도록 하겠습니다.
구체적인 적용 시기는 미정인 점 양해 부탁 드립니다.
감사합니다.

**Reply by Chordcode002** [reply]
감사합니다.
